### DISCUSSION TITLE
V1.2.0 Planning

### DISCUSSION THREAD

**Discussion by Raddelgo14** [Raddelgo14/Chat-Net-Pro#3] [Ideas]
Want help In V1.2.0 Planning, Will get credit for the planning.

**Comment by Raddelgo14** [comment]
Plan #1, Add a Apply system to make it so we have different jobs and that

**Comment by Raddelgo14** [comment]
Plan 2. Add a music control system
- New Music!
- No need to mute your tab, just turn off the music!

**Reply by Raddelgo14** [reply]
Update, Now all we need to add New Music

**Comment by GalaxiumNation** [comment]
Hey RbxDev, can you add this line of code to the Chat net and The admin chat for me?:
`   if (user.name == "Galaxium"){
     user.tag = "¯\_(ツ)_/¯";
   }`

Because I don't have my own custom name tag.
Thanks!
-Galaxium
